hles/transformation/src/main/scala/org/broadinstitute/monster/dap/dog/DietTransformations.scala

```diff
@@ -41,7 +41,7 @@ object DietTransformations {
       },
       dfEverMalnourished = rawRecord.getOptionalNumber("df_malnourished"),
       dfEverUnderweight = rawRecord.getOptionalNumber("df_underweight"),
-      dfEverOverweight = rawRecord.getOptionalNumber("df_overweight"),
+      dfEverOverweight = rawRecord.getOptionalNumber("df_overrweight"),
```

Comment:
any time i write code that intentionally includes a typo like this, i like to leave an explanatory comment. nothing fancy, just the software equivalent of "[sic]"